pkg/aws/aws.go

```diff
@@ -47,6 +51,7 @@ func GetCreds(svc stsiface.STSAPI, saml *okta.SAMLResponse, profile *config.Prof
 
 	resp, err := svc.AssumeRoleWithSAML(params)
 	if err != nil {
+		fmt.Println(fmt.Errorf("Failed to assume role for %s", profile.Name))
```

Comment:
`Errorf`'s not needed since you're not returning that created error.  `profile.Name `'s already a string so something more direct like:
```suggestion
		fmt.Println("Failed to assume role for " + profile.Name)
```